modules/serverless-autoscaling-maxscale-kn.adoc

```diff
@@ -5,7 +5,7 @@ You can use the `kn service` command with the `--max-scale` flag to create or mo
 
```

Comment:
I suspect you are updating existing content, so if you have time, please comment which assemblies this module is included in.